tests/test_responses_between_dates.py

```diff
@@ -0,0 +1,97 @@
+"""
+Tests for survey API endpoint that fetches all the user responses between two given dates 
+Test API response for different date ranges
+    - Date range whithin which all the test data is present
+    - Date range 1 year prior to the date where test data is stored
+    - Current date
+    - Date range covers first week of test data
+    - Date range covers first two weeks of test data
+    - Date range from a week prior to the dates where test data is present ,to the week succeeding the last date which has the test date
+"""
+import json
+import os
+import pytest
+from unittest.mock import patch
```

Comment:
Import order is not right. Look up pep-8 or put as a linter.
Remove modules that are not needed.

Reply:
Removed the unwanted imports and rearranged the order